shop_clothes/hw_shop_clothes/styles/_gallery.scss

```diff
@@ -0,0 +1,44 @@
+h1 {
```

Comment:
it's very general selector
are you aware that it will apply for ALL pages now, not only for gallery?